docs/source/changelog.rst

```diff
@@ -15,6 +15,7 @@ Changelog
         * Add source code links to API reference (:pr:`862`)
         * Update links for testing Dask/Spark integrations (:pr:`867`)
         * Update release documentation for featuretools (:pr:`868`)
+        * Fix README link to development (:pr:`872`)
```

Comment:
Can you start a new "Future Release" section above this one?

Reply:
Added